lib/services/android-project-service.ts

```diff
@@ -111,27 +110,35 @@ class AndroidProjectService implements IPlatformProjectService {
 	public interpolateData(projectRoot: string): IFuture<void> {
 		return (() => {
 			// Interpolate the activity name and package
-			var manifestPath = path.join(projectRoot, "AndroidManifest.xml");
-			var safeActivityName = this.$projectData.projectName.replace(/\W/g, '');
+			let manifestPath = path.join(projectRoot, "AndroidManifest.xml");
+			let safeActivityName = this.$projectData.projectName.replace(/\W/g, '');
 			shell.sed('-i', /__PACKAGE__/, this.$projectData.projectId, manifestPath);
 			shell.sed('-i', /__APILEVEL__/, this.getTarget(projectRoot).wait().split('-')[1], manifestPath);
 
-			var stringsFilePath = path.join(projectRoot, 'res', 'values', 'strings.xml');
+			let stringsFilePath = path.join(projectRoot, 'res', 'values', 'strings.xml');
 			shell.sed('-i', /__NAME__/, this.$projectData.projectName, stringsFilePath);
 			shell.sed('-i', /__TITLE_ACTIVITY__/, this.$projectData.projectName, stringsFilePath);
 			shell.sed('-i', /__NAME__/, this.$projectData.projectName, path.join(projectRoot, '.project'));
-
 		}).future<void>()();
 	}
 
 	public afterCreateProject(projectRoot: string): IFuture<void> {
 		return (() => {
-			var targetApi = this.getTarget(projectRoot).wait();
+			let targetApi = this.getTarget(projectRoot).wait();
 			this.$logger.trace("Android target: %s", targetApi);
 			this.runAndroidUpdate(projectRoot, targetApi).wait();
+			this.adjustMinSdk(projectRoot);
 		}).future<void>()();
 	}
 
+	private adjustMinSdk(projectRoot: string): void {
+		let manifestPath = path.join(projectRoot, "AndroidManifest.xml");
+		let apiLevel = this.getTarget(projectRoot).wait().split('-')[1];
+		if (apiLevel === "MNC") { // MNC SDK requires that minSdkVersion is set to "MNC"
+			shell.sed('-i', /android:minSdkVersion=".*"/, `android:minSdkVersion="${apiLevel}"`, manifestPath);
```

Comment:
should you include ? in the regex. If you have line android:minSdkVersion="L" android:smthElse="Value", the match will be everything up to Value".